deploy/cr.yaml

```diff
@@ -120,6 +137,20 @@ spec:
 
     configsvrReplSet:
       size: 3
+#      storage:
+#        engine: wiredTiger
+#        inMemory:
+#          engineConfig:
+#            inMemorySizeRatio: 0.9
```

Comment:
can we remove these 3 lines?

Reply:
https://github.com/percona/percona-server-mongodb-operator/pull/467/commits/a3d77d79a78b9eedca1a3d9b409f9101c4a427fb